html-tables/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Tables</title>
+</head>
+
+<body>
+  <h1>Student Grade Table</h1>
+  <table>
```

Comment:
make sure to include the `thead` and `tbody` elements into your table.

Reply:
![image](https://user-images.githubusercontent.com/90783131/151860152-0a6edfee-94d9-42c2-b10b-da095ff6e861.png)